server/fishtest/util.py

```diff
@@ -343,17 +343,25 @@ def is_active_sprt_ltc(run):
 
 
 def remaining_hours(run):
+    if "spsa" in run["args"]: # Tunes are frequently stopped short, and with "autoselect" may have too many games anyways
+        return 0
     r = run["results"]
+    N = sum(r["pentanomial"]) if "pentanomial" in r else r["wins"] + r["losses"] + r["draws"]
+    remaining_games = max(0, run["args"]["num_games"] - N)
     if "sprt" in run["args"]:
-        # current average number of games. Regularly update / have server guess?
-        expected_games = 58000
-        # checking randomly, half the expected games needs still to be done
-        remaining_games = expected_games / 2
-    else:
-        expected_games = run["args"]["num_games"]
-        remaining_games = max(0, expected_games - r["wins"] - r["losses"] - r["draws"])
-    game_secs = estimate_game_duration(run["args"]["tc"])
-    return game_secs * remaining_games * int(run["args"].get("threads", 1)) / (60 * 60)
+        average_games = 95000
+        llr = abs(run["args"]["sprt"]["llr"]) + 0.001
+        # Dumb estimate, very dumb near 0
```

Comment:
Possible div by zero in the next two lines.

Reply:
`rate` can't be zero because `llr` can't be zero, however yes `N` could be 0, oops